<?xml version="1.0"?>

<launch>
    <node name="interactive_pose_publisher" pkg="interactive_pose_publisher" type="interactive_pose_publisher_node">
    
        <rosparam command="load" file="$(find interactive_pose_publisher)/launch/initial_pose.yaml"/>

        <param name="interactive_pose_topic" value="set_pose"/>

        <param name="interactive_pose_parent" value="/world"/>
        <param name="interactive_pose_child" value="/marker"/>
    
    </node>

</launch>
